### PR TITLE
docs: add learn link to authz index page

### PR DESCRIPTION
**Pull request by marionschleifer**
### Description

As discussed in yesterday's meeting, adding a link to the authz learn tutorial from the authz index page on docs.

### Changelog

- [x] `CHANGELOG.md` is updated with user-facing content relevant to this PR. If no changelog is required, then add the `no-changelog-required` label.

### Affected components

- [x] Docs

### Affected page

https://deploy-preview-5387--hasura-docs.netlify.app/graphql/manual/auth/index.html

### CODE REVIEW

File: docs/graphql/manual/auth/authorization/index.rst [hunk 1]
```diff
@@ -46,6 +46,10 @@ auth service with GraphQL engine. You can just:
 
 Follow the example at :ref:`access control basics <auth_basics>`.
 
+Learn tutorial
+^^^^^^^^^^^^^^
+
+If you'd like to learn about access control by following a tutorial, check out our `auth learn course <https://hasura.io/learn/graphql/hasura-auth-slack/introduction/>`__.
```
Comment (sezgi) [inline docs/graphql/manual/auth/authorization/index.rst:52]:
```suggestion
If you'd like to learn about access control by following a tutorial, check out our Learn course, `Authentication with Hasura <https://hasura.io/learn/graphql/hasura-auth-slack/introduction/>`__.
```

**Comment by marionschleifer** [comment]
@sezgi I changed it to an admonition with the title "Learn course". Wouldn't make a separate heading because it fits nicely under the current one "Trying out access control" 🙂

**Comment by rikinsk** [comment]
@marionschleifer I have approved this. Though it might be worth thinking about the location of the note. the learn course is about authentication as well. Maybe similar notes might be helpful in the authentication index page or the authentication/authorization index page

**Comment by marionschleifer** [comment]
@rikinsk I was thinking the same. The reason I put it there is that the authentication part of the course is very brief at the end. And because it fits nicely in the "Try access control" section. Can we have the link in both pages maybe?

**Comment by marionschleifer** [comment]
@sezgi @rikinsk moved it to auth index for now so as to not duplicate it.

**Pull request merged**